src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -107,10 +108,11 @@ private IReadOnlyList<ProjectRuleSetConflict> FindConflicts(RuleSetInformation[]
 
         private RuleSetInformation[] GetAggregatedSolutionRuleSets()
         {
-            var solutionBinding = this.serviceProvider.GetService<ISolutionBindingSerializer>();
-            solutionBinding.AssertLocalServiceIsNotNull();
+            var configProvider = this.serviceProvider.GetService<IConfigurationProvider>();
+            configProvider.AssertLocalServiceIsNotNull();
 
-            BoundSonarQubeProject bindingInfo = solutionBinding.ReadSolutionBinding();
+            BoundSonarQubeProject bindingInfo = configProvider.GetBoundProject();
+            // TODO: CM2 - check that we are in legacy connected mode
```

Comment:
CM2?

Reply:
Short-hand for "Connected Mode version 2" - a marker to search to tag things I still need to do.